code/modules/vore/eating/belly_vr.dm

```diff
@@ -192,15 +192,18 @@
 	if(internal_contents.len && examine_messages.len)
 		var/formatted_message
 		var/raw_message = pick(examine_messages)
+		var/total_bulge = 0
 
 		formatted_message = replacetext(raw_message,"%belly",lowertext(name))
 		formatted_message = replacetext(formatted_message,"%pred",owner)
 		formatted_message = replacetext(formatted_message,"%prey",english_list(internal_contents))
 		for(var/mob/living/P in internal_contents)
-			if(!P.absorbed && P.size_multiplier >= bulge_size && bulge_size != 0) //This is required first, in case there's a person absorbed and not absorbed in a stomach.
-				return("<span class='warning'>[formatted_message]</span><BR>")
-			else if(P.absorbed || P.size_multiplier < bulge_size || bulge_size == 0) //Are they absorbed, too small to show up, or examining this stomach is disabled?
-				return ""
+			if(!P.absorbed) //This is required first, in case there's a person absorbed and not absorbed in a stomach.
+				total_bulge += P.size_multiplier
+		if(total_bulge >= bulge_size && bulge_size != 0)
+			return("<span class='warning'>[formatted_message]</span><BR>")
+		else if(total_bulge < bulge_size || bulge_size == 0) //Are they absorbed, too small to show up, or examining this stomach is disabled?
```

Comment:
You don't actually need the `if` in this. Just do `else` and return the empty string, since the only way to get to the `else` is with the conditions you want right there. There's no point in testing for them since we already know either total_bulge is less than bulge_size or bulge_size is 0